### PR TITLE
Send pageviews to storylines redis

### PR DESCRIPTION
**Pull request by TheoLeanse**

### CODE REVIEW

**Comment by TheoLeanse** [comment]
@commuterjoy

File: sinks/page-views.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+const logger = require('@financial-times/n-logger').default;
+const redis = require('redis');
+const denodeify = require('denodeify');
+
+const REDIS_URL = 'redis://x:08crztr6x7th40mh3hx7pggghkvnmtye@wise-daffodil-5327.redisgreen.net:11042/';
```
Comment (commuterjoy) [inline sinks/page-views.js]: move to config-vars and cycle this please

Comment (TheoLeanse) [inline sinks/page-views.js]: 👍

File: sinks/page-views.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+const client = redis.createClient({ url: REDIS_URL });
+const incrementView = denodeify(client.incr.bind(client));
+
+const getEvent = message => {
+	try {
+		const { ingest: { _body: { category, action } } } = message;
```
Comment (commuterjoy) [inline sinks/page-views.js]: can you use `egest.annotations.ingest.category` & `egest.annotations.ingest.action`

http://spoor-docs.herokuapp.com/annotations#ingest

**Comment by commuterjoy** [comment]
+1 looks good apart from the two comments

File: sinks/page-views.js (hunk 2 above)
Comment (TheoLeanse) [inline sinks/page-views.js]: thanks

**Pull request merged**